observability/telemetry/src/main/scala/com/daml/telemetry/OpenTelemetryOwner.scala

```diff
@@ -91,6 +92,19 @@ object OpenTelemetryOwner {
           )
         ),
       )
+      // timing buckets for gRPC server latency measurements with more precise granularity on latencies up to 5s
+      .registerView(
+        histogramSelectorWithRegex(
+          s"${DamlGrpcServerMetrics.GrpcServerMetricsPrefix}.*${OpenTelemetryTimer.TimerUnitAndSuffix}"
+        ),
+        explicitHistogramBucketsView(
+          Seq(
+            0.01d, 0.025d, 0.050d, 0.075d, 0.1d, 0.15d, 0.2d, 0.25d, 0.35d, 0.5d, 0.75d, 1d, 1.25d,
+            1.5d, 1.75d, 2d, 2.25d, 2.5d, 2.75d, 3d, 3.25d, 3.5d, 3.75d, 4d, 4.25d, 4.5d, 4.75d, 5d,
```

Comment:
Just wondering why a bucket every 250ms is needed between 1-5 seconds?

Reply:
Yeah it's a judgment call here tbh. I think a 0.5s gap between the reality and what is showed can be significant, so I added an in between with 250ms. Do you think it's too granular?

Reply:
Either way I will also update the documentation to show how these can be configured explicitly because I couldn't find it in the current docs at least

Reply:
I reduced it to every .5s between 3s and 5s to reduce the memory footprint